lib/sessionParticipants/index.js

```diff
@@ -59,19 +65,18 @@ SessionParticipants.prototype.getFirst = function(cb) {
 /**
  * Remove a Session Participant from the data structure.
  * 
- * @issuer {string}   The string as it was received in the SAML request/response
- * @sessionIndex {string}   The string as it was received in the SAML request/response. Only available in LogoutRequests 
- * @cb     {function} The callback that will be called with '(err, removedElement)'
+ * @serviceProviderId {string} The serviceProviderId value of the session participant
+ * @sessionIndex {string} The sessionIndex of the session participant
+ * @nameId {string} The nameId of the session participant
+ * @cb {function} The callback that will be called with '(err, removedElement)'
  */
```

Comment:
No functionality change here but it seems that there was some copy-pasting from the `get` method above and some of the method arguments and comments did not make sense with how the function was being used.